tests/.gitignore

```diff
@@ -0,0 +1,767 @@
+!install/
```

Comment:
This file can be simplified significantly. For example, we don't need the Node stuff.